lhotse/recipes/mtedx.py

```diff
@@ -0,0 +1,237 @@
+"""
+MTEDx is a collection of transcribed and translated speech corpora:
+ https://openslr.org/100
+
+It has 8 languages:
+es - 189h
+fr - 189h
+pt - 164h
+it - 107h
+ru - 53h
+el - 30h
+ar - 18h
+de - 15h
+
+A subset of this audio is translated and split into the following partitions:
+     - train
+     - dev
+     - test
+     - iwslt2021 sets
+
+This recipe only prepares the ASR portion of the data.
+"""
+import logging
+import re
+from collections import defaultdict
+from typing import Dict, Iterable, List, Optional, Union
+from pathlib import Path
+
+from cytoolz import sliding_window
+
+import tarfile
+import shutil
+from lhotse import Recording, RecordingSet, SupervisionSegment, SupervisionSet, validate_recordings_and_supervisions
+from lhotse.qa import remove_missing_recordings_and_supervisions, trim_supervisions_to_recordings
+from lhotse.utils import Pathlike, urlretrieve_progress
+import regex as re2
+from functools import partial
+import unicodedata
```

Comment:
I don't think `unicodedata` and `regex` are direct lhotse's dependencies -- could you use `lhotse.utils.is_module_available` to check that they are installed raise an error saying to install them otherwise?

Reply:
I have used importlib in the past to do this. You just try to import and if it fails you can install with pip. Is that ok with you?

Reply:
I think `unicodedata` is part of the [Python Standard Library](https://docs.python.org/3/library/unicodedata.html). `regex` is a third-party library, so perhaps it may be better to replace it with `re` (unless there is some functionality not available in `re`)?

Reply:
I think python's re doesn't know unicode character classes  -- e.g. ``\p{Zs}``

Reply:
Yenda is right, and I used it just to be able to to \p{Zs} in fact.

Reply:
It's fine -- just add a check like this for example: https://github.com/lhotse-speech/lhotse/blob/master/lhotse/features/librosa_fbank.py#L85-L90